pipeline/recon/web/gobuster.py

```diff
@@ -59,21 +59,14 @@ class GobusterScan(luigi.Task):
     threads = luigi.Parameter(default=defaults.get("threads"))
     wordlist = luigi.Parameter(default=defaults.get("gobuster-wordlist"))
     extensions = luigi.Parameter(default=defaults.get("gobuster-extensions"))
+    requirements = ["recursive-gobuster", "gobuster"]
```

Comment:
unless it's handled somewhere else, go is a requirement as well

Reply:
This was an oversight, corrected but pipeline fails due to Go version mismatch